python/katana/plugin.py

```diff
@@ -56,13 +56,13 @@ def load_module(self, fullname):
 
         assert fullname in self._insert_table
 
-        # This will get the module from sys.modules if it is already loaded.
-        module = importlib.import_module(self._insert_table[fullname])
+        aliased_name = self._insert_table[fullname]
+        if aliased_name in sys.modules:
+            module = sys.modules[aliased_name]
+        else:
+            module = importlib.import_module(aliased_name)
         if fullname in sys.modules:
```

Comment:
should this statement be only under "else"?

Reply:
It can be. I'll move it.